exopy_pulses/tasks/tasks/instrs/transfer_sequence_task.py

```diff
@@ -95,6 +97,50 @@ def perform(self):
 
         for k, v in infos.items():
             self.write_in_database(k, v)
+            
+    def compile_and_plot(self, variables):
```

Comment:
I am not sure this belongs on the task. I feel like it would be better on the plugin.